### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by UserNotFound**
Since Alpine doesn't have packages for the mainline (1.15.x) nginx version, we'll switch this repo to use the same installation method as the https://github.com/aptible/docker-nginx.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,15 @@
-FROM alpine:3.6
+FROM alpine:3.8
 
 # This can be constant and embedded in the Docker image: it just needs to be
 # big enough.
-RUN apk add --no-cache openssl \
+RUN apk add --no-cache openssl>=1.0.2k \
```
Comment (krallin) [inline Dockerfile]: Can you clarify why this is necessary? AFAICT, the only `openssl` version available in `3.6` is `1.0.2q`, so what's up there?

File: install-nginx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+BUILD_DIR="$(mktemp -d)"
+
+cd "$BUILD_DIR"
+
+curl -fsSL "http://nginx.org/download/${NGINX_ARCHIVE}" -o "$NGINX_ARCHIVE"
+echo "${NGINX_SHASUM}  ${NGINX_ARCHIVE}" | sha1sum -c -
+tar zxf "$NGINX_ARCHIVE"
+
+curl -fsSL "https://github.com/simpl/ngx_devel_kit/archive/v${NGX_DEVEL_KIT_VERSION}.tar.gz" -o "$NGX_DEVEL_KIT_ARCHIVE"
```
Comment (krallin) [inline install-nginx]: This is a toolkit to build extensions, which we're not using here... I'm pretty sure you can remove this.

File: install-nginx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+tar zxf "$NGX_DEVEL_KIT_ARCHIVE"
+
+echo "Downloaded:"
+ls -l
+
+cd "$NGINX_RESOURCE"
+
+# Cribbing from
+# http://git.alpinelinux.org/cgit/aports/tree/main/nginx/APKBUILD
+# but adding --with-http_realip_module to support "set_real_ip_from" directive
```
Comment (krallin) [inline install-nginx]: We're only doing TCP forwarding and TLS termination, we definitely don't need `--with-http_realip_module`.

File: install-nginx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  --http-fastcgi-temp-path=/tmp/nginx/fastcgi \
+  --user=nginx \
+  --group=nginx \
+  --with-pcre-jit \
+  --with-http_ssl_module \
+  --with-http_gzip_static_module \
+  --with-http_realip_module \
+  --with-stream \
+  --with-stream_ssl_module \
+  --with-stream_ssl_preread_module
```
Comment (krallin) [inline install-nginx]: All the HTTP stuff here can probably be removed

File: Dockerfile (hunk 1 above)
Comment (UserNotFound) [reply Dockerfile]: I was just mirroring the configuration from docker-nginx, but I agree that this is fairly useless at this time.

File: install-nginx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+NGINX_VERSION='1.15.6'
+NGINX_SHASUM='b644566068d2995533629ac1329a296ea135a69b'
+NGINX_RESOURCE="nginx-${NGINX_VERSION}"
+NGINX_ARCHIVE="${NGINX_RESOURCE}.tar.gz"
+
+NGX_DEVEL_KIT_VERSION="0.3.0"
+NGX_DEVEL_KIT_SHASUM="b556d068db23037be30436af559795f45dd93c67"
+NGX_DEVEL_KIT_RESOURCE="ngx_devel_kit-${NGX_DEVEL_KIT_VERSION}"
+NGX_DEVEL_KIT_ARCHIVE="${NGX_DEVEL_KIT_RESOURCE}.tar.gz"
```
[the remark this answers is not in the document]
Comment (UserNotFound) [reply install-nginx]: Thanks!

**Comment by krallin** [comment]
@UserNotFound thanks for making those changes — please proceed to merging

**Pull request merged**